pages/users.js

```diff
@@ -1,46 +1,45 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from 'react';
 import withRedux from 'next-redux-wrapper';
 import configureStore from '../store/configureStore';
-import checkTangleUsers from '../actions/checkTangleUsers';
-import fetchUserList from '../actions/fetchUserList';
-// import Layout from '../layouts/Main'
 import Layout from '../layouts/material/Main';
 import UserList from '../components/UserList';
 
-const UsersPage = (props) => {
-  const { userList } = props;
-  return (
-    <Layout>
-      <h2>Users</h2>
-      <UserList users={userList} />
-    </Layout>
-  );
-};
+class UsersPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userList: [],
+    };
+  }
 
-UsersPage.propTypes = {
-  userList: PropTypes.array,
-};
+  componentDidMount() {
+    fetch('/api/fetchUserList')
```

Comment:
How can we extend routing path? We should have an explicit list of `/api/` which points to actual response.